src/modules/update/Kconfig

```diff
@@ -0,0 +1,12 @@
+menuconfig USE_AUTO_UPDATE
+    bool "Auto Update Options"
```

Comment:
I have no idea why you added "auto" in here :-D makes no sense.

I'd keep it only "update", or call it "system update" and change everywhere.